src/Razor/Microsoft.NET.Sdk.Razor/test/ConcatenateFilesTest.cs

```diff
@@ -382,14 +383,14 @@ public void BundlesScopedCssFiles_UpdatesBundleWhenContentsChange()
                     }),
             };
 
+            Thread.Sleep(TimeSpan.FromSeconds(1));
```

Comment:
I'm cool with adding delays. But could we use `await Task.Delay(...)` instead? That way we can yield to the scheduler rather than block.

Reply:
I spent too long figuring out why it didn't like the `Task` return type on this method only to realize it was resolving to an MSBuild Task type instead of `System.Threading.Tasks.Task`.

🤦